src/explore-education-statistics-common/src/services/redirectService.ts

```diff
@@ -10,11 +12,14 @@ interface Redirect {
   toSlug: string;
 }
 
-const contentApiUrl = process.env.CONTENT_API_BASE_URL;
+export const redirectPathStarts = {
+  methodologies: '/methodology',
+  publications: '/find-statistics',
+};
 
 const redirectService = {
   async list(): Promise<Redirects> {
-    return (await fetch(`${contentApiUrl}/redirects`)).json();
```

Comment:
Eurgh, so apparently this causes build errors in the frontend. We could re-implement the frontend version of this service and have two, but that's messy. Problem as things stand I need to make this call available from `ContentHtml` which is in common. 

@ntsim  I'm gonna hold off solving this pending your thoughts on my big TODO: comment in `AnchorHtml`, as I suspect there might be a few changes 
![image](https://github.com/dfe-analytical-services/explore-education-statistics/assets/63285990/5e2fc662-67a8-45f3-895d-3ff808f9db7b)